src/views/navbar/style.js

```diff
@@ -346,3 +384,17 @@ export const MarkAllSeen = styled.span`
       props.isActive ? props.theme.brand.default : props.theme.text.alt};
   }
 `;
+
+// We make it a real link element because anchor links don’t work properly with React Router.
+// Ref: https://github.com/ReactTraining/react-router/issues/394.
+export const SkipLink = Tab.withComponent('a').extend`
```

Comment:
I didn’t find any occurence of `.extend` (or `withComponent(…).extend`) in your codebase, so I assume you might not like this for whatever reason? 😄

Reply:
We've been using styled-components since before that was a thing, LGTM!